AndroidSampleAppWithSdk/app/build.gradle

```diff
@@ -87,21 +94,17 @@ dependencies {
     implementation "com.squareup.retrofit2:converter-jackson:$retrofitVersion"
     implementation "com.squareup.retrofit2:adapter-rxjava2:$retrofitVersion"
 
-    //RxJava
-    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
-    implementation 'io.reactivex.rxjava2:rxjava:2.2.19'
-
     // Dependency Injection
-    def daggerVersion = '2.25.4'
+    def daggerVersion = '2.27'
     implementation "com.google.dagger:dagger:$daggerVersion"
 
-    implementation 'sh.locus.lotr:lotr-javasdk:0.0.1-20200824.122945-261'
+    implementation 'sh.locus.lotr:lotr-javasdk:0.0.1-20210331.050530-358'
```

Comment:
this will come from maven central itself?

Reply:
This is a commented out portion which I use for local builds only. (I want to test release build with obfuscation before publishing anything on maven central.)